cypress/integration/brewery/ScenarioManager-test.spec.js

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Cosmo Tech.
```

Comment:
@nborde-CSM @csm-thu  A question about naming: do we still add "test" to file names ? I thought we decided not to add it because all files in this folder are actually tests :slightly_smiling_face: